src/test/ui/consts/too_generic_eval_ice.stderr

```diff
@@ -0,0 +1,47 @@
+error[E0599]: no associated item named `HOST_SIZE` found for type `Foo<A, B>` in the current scope
+  --> $DIR/too_generic_eval_ice.rs:7:19
+   |
+LL | pub struct Foo<A, B>(A, B);
+   | --------------------------- associated item `HOST_SIZE` not found for this
+...
+LL |         [5; Self::HOST_SIZE] == [6; 0]
+   |                   ^^^^^^^^^ associated item not found in `Foo<A, B>`
+   |
+   = note: the method `HOST_SIZE` exists but the following trait bounds were not satisfied:
+           `A : std::marker::Sized`
+           `B : std::marker::Sized`
```

Comment:
that's a different bug. This regression test is to prevent the ICE from recurring. The fact that what's done in the test is unsupported by rustc is likely lazy normalization (allowing array length constants to refer to generics of the surrounding scope)

Reply:
(And it should stay unsupported on stable without T-Lang approval.)